feat(crons): Add product docs for environments (#6679)
getsentry/sentry-docs: src/docs/product/security-policy-reporting.mdx

`sentry_environment`

> : The environment name (e.g. production). The environment name can't contain newlines, spaces, or forward slashes, can't be the string "None", or exceed 64 characters. Also, environments are case sensitive.

Reviewer: ```suggestion
: The environment name (for example, production). 
The environment name is case sensitive, can't contain new lines, spaces, or forward slashes. It can't be the string "None" or exceed 64 characters. 
```
